### PR TITLE
Automation for CEPH-83613272

### PR DESCRIPTION
**Pull request by aarsharm**
Pass logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YUZTQK
Polarion ID: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83613272
[CEPH-83613272 - Verify group mirroring for images with 0 size & Verify group mirroring with 0 images in group](https://polarion.engineering.redhat.com/polarion/redirect/project/CEPH/workitem?id=CEPH-83613272)

### CODE REVIEW

File: tests/rbd_mirror/test_group_mirror_with_zero_size_image.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+                group_image_list_secondary
+            ):
+                raise Exception(
+                    "Group image list does not match for primary and secondary cluster"
+                )
+            log.info(
+                "Successfully verified image list for the group matches across both cluster"
+            )
+
+            # Verify group mirroring status on both clusters & Match global id of both cluster
```
Comment (sunilangadi2) [inline tests/rbd_mirror/test_group_mirror_with_zero_size_image.py:185]: if image is `zero` size don't having any data then from line 185 remaining test doesn't make any sense, I feel we can remove and save time for the execution.

**Review by sunilangadi2 — CHANGES_REQUESTED** [review]

**Review by chaitanyadommeti — APPROVED** [review]
LGTM

File: tests/rbd_mirror/test_group_mirror_with_zero_size_image.py (hunk 1 above)
Comment (aarsharm) [reply tests/rbd_mirror/test_group_mirror_with_zero_size_image.py:185]: even if image is 0 size, group should still be created on secondary and group status should still show up+replaying on secondary. This test is to verify EDGE and boundary case that if image is 0 size, image and group should still replicate to secondary with no error in group status

**Review by sunilangadi2 — APPROVED** [review]

**Review by Manohar-Murthy — APPROVED** [review]
lgtm.
